This file is part of MXE.
See index.html for further information.

From 29c2441e39372b34dfac348954385d2fecb361d1 Mon Sep 17 00:00:00 2001
From: Daniel Stenberg <daniel@haxx.se>
Date: Mon, 6 Aug 2012 15:04:25 +0200
Subject: [PATCH] gtls: fix build failure by including nettle-specific headers

Bug: http://curl.haxx.se/bug/view.cgi?id=3554668
Reported by: Anthony G. Basile
(cherry picked from commit 986c7949c0638bebb99bd08d63216433254c2185)

diff --git a/lib/gtls.c b/lib/gtls.c
index d981ef1..c750a6f 100644
--- a/lib/gtls.c
+++ b/lib/gtls.c
@@ -34,7 +34,10 @@
 
 #include <gnutls/gnutls.h>
 #include <gnutls/x509.h>
-#ifndef USE_GNUTLS_NETTLE
+#ifdef USE_GNUTLS_NETTLE
+#include <gnutls/crypto.h>
+#include <nettle/md5.h>
+#else
 #include <gcrypt.h>
 #endif
 
-- 
1.7.10.4

